projects/kit/src/lib/masks/date-time/utils/parse-date-time-string.ts

```diff
@@ -3,14 +3,15 @@ import {DATE_TIME_SEPARATOR} from '../constants';
 export function parseDateTimeString(
     dateTime: string,
     dateModeTemplate: string,
+    dateTimeSeparator = DATE_TIME_SEPARATOR,
 ): string[] {
```

Comment:
Let's improve signature of this utility to the following way:

```suggestion
export function parseDateTimeString(
    value: string,
    {
        dateModeTemplate,
        dateTimeSeparator,
    }: {
        dateModeTemplate: string;
        dateTimeSeparator: string;
    },
): [date: string, time: string] {
```

Reply:
**Usage:**

```ts
const [date, time] = parseDateTimeString(value, {
    dateModeTemplate,
    dateTimeSeparator,
});
```